lib/dal/src/func/binding/management.rs

```diff
@@ -95,6 +95,7 @@ impl ManagementBinding {
                                 kind: {json_name},
                                 properties?: {sv_type},
                                 geometry?: {{ [key: string]: Geometry }},
+                                incomingConnections?: {{ [socket: string]: {INCOMING_CONNECTION_TYPE} | {INCOMING_CONNECTION_TYPE}[] }},
                                 connect?: {{
```

Comment:
I don't think this is for your review at all, but ... `connect` is an operation, not an input, right? Like, it seems like this is only used in `type Input`, so it's weird we have this here. I guess it technically passes typecheck since we always pass undefined, but still.